src/test/java/samples/CDExample.java

```diff
@@ -0,0 +1,54 @@
+package samples;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CDExample {
+    public String text;
+    public List<String> weaselWordsList = Arrays.asList("many", "various", "very", "fairly", "several", "extremely",
+            "exceedingly", "quite", "remarkably", "few", "surprisingly", "mostly", "largely", "huge",
+            "tiny", "is a number", "are a number", "excellent", "interestingly", "significantly",
+            "substantially", "clearly", "vast", "relatively", "completely");
+
+    public static void main(){
+        CDExample inst = new CDExample();
+        inst.cleanText();
+    }
+
+    public void cleanText(){
+        if (text != null && hasWeaselWords()) { //LEFT
+            removeWeaselWords();
+            removeDuplicateWords(); //RIGHT
+        }
+    }
+
+    public boolean hasWeaselWords(){
+        for (String word : text.split(" ")) {
+            for (String s : weaselWordsList) {
+                if (s.equals(word)) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    public void removeWeaselWords(){
+        for (String s : weaselWordsList) {
+            text.replace(s, "");
+        }
+    }
+
+    public void removeDuplicateWords(){
```

Comment:
@galilasmb isso ainda está bugado, não? chegasse a testar? com que entradas?

Reply:
Fiz alguns testes, funcionou. Exemplo:

Entrada: the the dog
Saída: the dog

Entrada: isso tá bugado, não tá?
Saída: isso tá bugado, não tá?

Entrada: the the dog the dog dog
Saída: the dog the dog

Reply:
@galilasmb testa com strings que tenham mais de um espaço em branco entre duas palavras para ver o que acontece